bayes/inference_problem.py

```diff
@@ -218,9 +255,30 @@ def prior_MVN(self):
                 raise RuntimeError(
                     f"You defined {name} as latent but did not provide a prior distribution!."
                 )
-            mean, sd = self.prm_prior[name]
+            if self.prm_prior[name].dist.name != "norm":
+                raise RuntimeError(
+                    f"VB problem can only handle normal priors, you provided `{self.prm_prior[name].dist.name}` for parameter `{name}`."
+                )
+
+            mean, var = self.prm_prior[name].mean(), self.prm_prior[name].var()
             for _ in range(latent.N):
                 means.append(mean)
-                precs.append(1.0 / sd ** 2)
+                precs.append(1.0 / var)
+
+        return MVN(
```

Comment:
Wouldn't it be logic to fully remove the self implemented distributions, so return a scipy distribution would then also be possible?

Reply:
Good idea, I'll have a look at it. Minor issues are 
* `scipy.stats.gamma` does not allow access to shape and scale directly, but they must be calculated via e.g. mean and variance
* `scipy.stats.multivariate_normal` uses the covariance instead of precision, so we need to compute the inverse at some point.

Nothing unsolvable.

Reply:
As for accessing the shape and scale in frozen distributions
https://stackoverflow.com/questions/37501075/how-to-get-parameter-arguments-from-a-frozen-spicy-stats-distribution
Related to the mvn, it seems that there are more user requests, not sure what do do right now, since for large matrices (in the posterior) inverting the matrix might be significant. Even though we do that in VB anyhow, right?
https://github.com/numpy/numpy/issues/17158

Reply:
We currently use pass the names to the MVN which is totally convenient in all my examples. As I said, I think its a good idea, but it need a bit of planning. #43